lib/tasks/data/sanitize_data.rake

```diff
@@ -0,0 +1,56 @@
+require 'highline'
+
+namespace :ofn do
+  namespace :data do
+    desc 'Sanitize data'
```

Comment:
Sanitize can also mean to discard input data which doesn't comply to requirements. So if someone enters an invalid email address into a form. But it's also used for removing classified data. Maybe a native speaker could tell us if this name is good? I'm okay with it.

Reply:
Yeah, you are right. I have just renamed it to anonymize :-)
I think they are both right, actually:
![Screenshot 2019-03-20 at 11 51 37](https://user-images.githubusercontent.com/1640378/54682341-8ed93f80-4b06-11e9-865b-74aa09ad106b.png)